### DISCUSSION TITLE
New event action: create and assign virtual folder

### DISCUSSION THREAD

**Discussion by mbendix** [drakkan/sftpgo#1329] [Ideas]
Hello drakkan,

I miss a event actions to ...
- create a new folder
- assign a virtual folder to a user

Use case:
E.g. simple file-exchange with clients: Each client has an own user with inbox and outbox directories. On our site we have an import-user who has access to each client inbox, to obtain the delivered files.

When we create a new user for a client, the create user provider event is triggered and we have already an event action to create the in- and outbox on the file system. So I would like to extend this event rule with a new event action(s) to create and assign automatically a virtual folder for a specific user.

Best regards,
Mark

**Comment by drakkan** [comment]
Hello,

this could be done, maybe in a different/easier way. Do you want to support the project and sponsor this feature? Thank you
